src/VisualStudio/CSharp/Test/PersistentStorage/AbstractPersistentStorageTests.cs

```diff
@@ -79,8 +79,6 @@ protected AbstractPersistentStorageTests()
         public void Dispose()
         {
             // This should cause the service to release the cached connection it maintains for the primary workspace
-            _persistentLocationService?.RaiseShutdown();
```

Comment:
no more complex shutdown/eventing for what is effectively the concept "where should i store the cache db across vs sessions?"